tests/nvmeof/test_ceph_nvmeof_gateway.py

```diff
@@ -21,8 +21,8 @@
 def configure_subsystems(rbd, pool, gw, config):
     """Configure Ceph-NVMEoF Subsystems."""
     sub_nqn = config["nqn"]
-    max_ns = config.get("max_ns", 32)
-    gw.create_subsystem(sub_nqn, config["serial"], max_ns)
+    max_ns = {"max-namespaces": config.get("max_ns", 32)}
```

Comment:
probably this should be replaced everywhere? then

Reply:
I guess from now on we do not need to include max-namespaeces in test module or suite, it should be directly passed from test suite, right? Refer - https://github.com/red-hat-storage/cephci/blob/master/ceph/nvmeof/nvmeof_gwcli.py#L105

Reply:
@rahullepakshi 
Do you mean we want to send max-namespaces also one of the arg from testsuite

Reply:
@HaruChebrolu IMO, yes. But if it does not hold good for this test scenario. then we can as above
@sunilkumarn417 WDYT?

Reply:
IMO, for this scenario, it would be good if we pass `max-namespaces` as kwargs(as its an optional argument for this command). May be for other testcases where max-namespaces is acting as a configurable parameter, it makes sense to have as a parameter from test-suite itself.